deployment/CP05MOAS-GL336_Deploy.csv

```diff
@@ -47,9 +47,9 @@ AR-34,,,,CP05MOAS-GL336-02-FLORTM000,8,1,2019-04-08T00:14:00,2019-04-08T10:56:00
 AR-34,,,,CP05MOAS-GL336-03-CTDGVM000,8,1,2019-04-08T00:14:00,2019-04-08T10:56:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-CTDGVM-09060,39.95807,-71.00037,,1000,1000,FZ-2
 AR-34,,,,CP05MOAS-GL336-04-DOSTAM000,8,1,2019-04-08T00:14:00,2019-04-08T10:56:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-DOSTAM-00542,39.95807,-71.00037,,1000,1000,FZ-2
 AR-34,,,,CP05MOAS-GL336-05-PARADM000,8,1,2019-04-08T00:14:00,2019-04-08T10:56:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-PARADM-50163,39.95807,-71.00037,,1000,1000,FZ-2
-TS20190516,,,,CP05MOAS-GL336-00-ENG000000,9,1,2019-05-16T23:37:11,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGCON-GLDRCP-00336,40.03682,-70.7816,,1000,1000,FZ-1
-TS20190516,,,,CP05MOAS-GL336-01-ADCPAM000,9,1,2019-05-16T23:37:11,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-ADCPAM-00543,40.03682,-70.7816,,1000,1000,FZ-1
-TS20190516,,,,CP05MOAS-GL336-02-FLORTM000,9,1,2019-05-16T23:37:11,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-FLORTM-03206,40.03682,-70.7816,,1000,1000,FZ-1
-TS20190516,,,,CP05MOAS-GL336-03-CTDGVM000,9,1,2019-05-16T23:37:11,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-CTDGVM-09060,40.03682,-70.7816,,1000,1000,FZ-1
-TS20190516,,,,CP05MOAS-GL336-04-DOSTAM000,9,1,2019-05-16T23:37:11,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-DOSTAM-00542,40.03682,-70.7816,,1000,1000,FZ-1
-TS20190516,,,,CP05MOAS-GL336-05-PARADM000,9,1,2019-05-16T23:37:11,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-PARADM-50163,40.03682,-70.7816,,1000,1000,FZ-1
\ No newline at end of file
+TS20190516,,,,CP05MOAS-GL336-00-ENG000000,9,1,2019-05-16T23:37:11,2019-07-28T00:38:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGCON-GLDRCP-00336,40.03682,-70.7816,,1000,1000,FZ-1
+TS20190516,,,,CP05MOAS-GL336-01-ADCPAM000,9,1,2019-05-16T23:37:11,2019-07-28T00:38:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-ADCPAM-00543,40.03682,-70.7816,,1000,1000,FZ-1
+TS20190516,,,,CP05MOAS-GL336-02-FLORTM000,9,1,2019-05-16T23:37:11,2019-07-28T00:38:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-FLORTM-03206,40.03682,-70.7816,,1000,1000,FZ-1
+TS20190516,,,,CP05MOAS-GL336-03-CTDGVM000,9,1,2019-05-16T23:37:11,2019-07-28T00:38:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-CTDGVM-09060,40.03682,-70.7816,,1000,1000,FZ-1
+TS20190516,,,,CP05MOAS-GL336-04-DOSTAM000,9,1,2019-05-16T23:37:11,2019-07-28T00:38:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-DOSTAM-00542,40.03682,-70.7816,,1000,1000,FZ-1
+TS20190516,,,,CP05MOAS-GL336-05-PARADM000,9,1,2019-05-16T23:37:11,2019-07-28T00:38:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-PARADM-50163,40.03682,-70.7816,,1000,1000,FZ-1
```

Comment:
Maybe you should add a note explaining the glider 'recover' time since it wasn't actually recovered.  That would help explain why there is no recover cruise ID.

Reply:
I concur with Rebecca.  Please add a note to line 50 noting that the glider was lost.  Perhaps put "N/A" in the recovery cruise column so that we know that space is intentionally left blank.